src/development/ssh.md

```diff
@@ -137,6 +137,15 @@ $ ssh-add path-to-your-key
 
 4.  Try again.
 
+### Try altering your SSH configuration
+
+You can try to append some directives to your `~/.ssh/config` file as in the example below, where `id_platformsh` is the public key file.
+
+    Host platform.sh
+    IdentityFile ~/.ssh/id_platformsh
+
+With this configuration, you should be asked for a password if your key has one.
+
```

Comment:
I'm not sure about this addition.  "Altering" isn't a good description here of what's being proposed.  I'm also not sure why explicitly identifying the key would make a difference.  If it does, the reason for that should be explained.

Reply:
Added explanation about OpenSSH (not mentioned explicitly) not looking at non-default key file names (based on man ssh).